### PR TITLE
Small Refactorings

### PR DESCRIPTION
**Pull request by Bonczidai**
Signed-off-by: Bonczidai Levente levente.bonczidai@hpe.com

### CODE REVIEW

File: test/io/cloudslang/chef/test_chef_bootstrap_run.inputs.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #
 ####################################################
 test_chef_bootstrap_run_success:
-  testSuites: [chef]
+  testSuites: [chef-local]
```
Comment (Bonczidai) [inline test/io/cloudslang/chef/test_chef_bootstrap_run.inputs.yaml]: first we are going to test it locally (and then automate if possible) - changed to be easier to find which test suites are note automated yet

**Comment by Bonczidai** [comment]
Hi @benc-uk 

I made some small refactorings regarding your CloudSlang/cloud-slang-content#347 PR so they will respect the style in other flows -- please review them

1 convention we usually apply: we don't specify **empty** default values for **non-required** inputs

e.g. when you have

```yaml
- non_req_input:
     default: "''" # just delete this line and handle the null check in flow logic (if necessary)
     required: false
```

**Comment by benc-uk** [comment]
Most of these look fine.

I tried very hard to get the Chef SSH operations to work without the `default: "''"` but it never worked, the SSH operation always complained about missing inputs. I can try again with these changes but I think the tests will fail

**Comment by Bonczidai** [comment]
let us know if it fails and we'll investigate it :wink:

**Pull request merged**